flipt-client-java/src/main/java/io/flipt/client/FliptEvaluationClient.java

```diff
@@ -348,6 +352,31 @@ public ArrayList<Flag> listFlags() throws EvaluationException {
     return resp.getResult().get();
   }
 
+  /**
+   * setSnapshot sets the snapshot for the Flipt client.
+   *
+   * @param snapshot the snapshot to set
+   */
+  public Result<String> setSnapshot(String snapshot) throws EvaluationException {
```

Comment:
returning the result was just for debugging, will remove this and make it throw an exception if result is not successful